### DISCUSSION TITLE
"List of all available images" link on the VNM's readme file

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#134] [Q&A] [answered]
On the readme of VNM, there is the following information:

> List of all available images:
> https://github.com/NeuroDesk/caid/packages

Do we have to mention the complete list of packages (even if they are not included in NeuroDesk)?
Or do we have this list only because it is nicer to browse?
Is this list really different from the list of packages included in NeuroDesk?

**Comment by stebo85** [comment, marked as answer]
Dear Oren,

The https://github.com/NeuroDesk/caid/packages list contains all containers that were ever build for the project, but what's included in the current release is controlled by this: https://github.com/NeuroDesk/neurodesk/blob/master/neurodesk/apps.json 

Do we have to mention the complete list of packages (even if they are not included in NeuroDesk)?
no - we can remove this link if everyone agrees that it's too much information - what does everyone else think?

Or do we have this list only because it is nicer to browse?
correct - it's genuinely nice to browse

Is this list different from the list of packages included in NeuroDesk?
yes, this is the list of things that are included: https://github.com/NeuroDesk/neurodesk/blob/master/neurodesk/apps.json

**Reply by aswinnarayanan** [reply]
I think both lists can be pretty useful. Currently the neurocontainers repo and package list aren't as discoverable as vnm, and people may want specific versions of containers.

@civier, did someone point out the package list as being confusing or too much information? We could amend the description to make this clearer.
